spring-cloud-starter-single-step-batch-job/src/main/java/org/springframework/cloud/task/batch/autoconfigure/SingleStepJobAutoConfiguration.java

```diff
@@ -78,7 +78,7 @@ private void validateProperties(SingleStepJobProperties properties) {
 
 	@Bean
 	@ConditionalOnMissingBean
-	@ConditionalOnProperty(prefix = "spring.batch.job", name = "jobName")
```

Comment:
I think it should be sufficient to change this line to
`@ConditionalOnProperty(prefix = "spring.batch.job", name = "job-name")`
If `name` is given in kebab case, then it will also match camel case. But not the other way around.

Reply:
@hpoettker great catch!   Thanks!